src/main/java/com/ajou/hertz/domain/administrative_area/entity/Sido.java

```diff
@@ -0,0 +1,32 @@
+package com.ajou.hertz.domain.administrative_area.entity;
+
+import com.ajou.hertz.common.entity.TimeTrackedBaseEntity;
+import com.ajou.hertz.domain.administrative_area.constant.SidoList;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Entity
+@Table(name = "adminitrative_area_sido")
+public class Sido extends TimeTrackedBaseEntity{
```

Comment:
```suggestion
public class AdminiStrativeAreaSido extends TimeTrackedBaseEntity{
```

좀 더 길어지긴 하겠지만, 구체적인 이름을 사용하는 편이 좋아보입니다.